packages/island/src/theme-default/components/BackDrop/index.tsx

```diff
@@ -0,0 +1,13 @@
+import styles from './index.module.scss';
+import type { ComponentPropsWithIsland } from 'islandjs';
+interface Props {
```

Comment:
It should have a new line after the last import statement.

Reply:
okok